SteamReview/SteamReview/Steam.cpp

```diff
@@ -290,8 +322,10 @@ bool FSteam::DeleteACategory()
 		std::cout << "Selected the category to delete" << std::endl;
 		int CategorySelected;
 		std::cin >> CategorySelected;
+		ValidateInput(CategorySelected, 1, CategoryContainer.GetCurrentCategoryAmount());
 
-		CategorySelected = ValidateInput(CategorySelected, 1, CategoryContainer.GetCurrentCategoryAmount()) - 1;
+		//Subtract 1 to select the current object inside of the array
+		CategorySelected = CategorySelected - 1;
 		return CategoryContainer.DeleteCategory(CategorySelected);
 	}
 	return false;
```

Comment:
Oh it all makes sense on why you have that ComeBackMainMenu function, I'm still not fully convinced by the name though, as it is not clear that it will require some interaction with the user